pkgs/by-name/ka/kakoune-lsp/package.nix

```
@@ -1,34 +1,37 @@
-{ stdenv, lib, fetchFromGitHub, rustPlatform, perl, CoreServices, Security, SystemConfiguration }:
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, substituteAll, perl
+, stdenv, CoreServices, Security, SystemConfiguration
+}:
 
 rustPlatform.buildRustPackage rec {
   pname = "kakoune-lsp";
-  version = "16.0.0";
+  version = "17.0.1";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-d4Tc6iYp20uOKMd+T2LhWgXWZzvzq1E+VWqjhhiIiHE=";
+    sha256 = "sha256-uXKquAjfytUn/Q0kx+0BGRQTkVMQ9rMRnTCy622upag=";
   };
```

- https://github.com/NixOS/nixpkgs/issues/277994
- https://github.com/NixOS/nixpkgs/issues/310373

```suggestion
  src = fetchFromGitHub {
    owner = "kakoune-lsp";
    repo = "kakoune-lsp";
    rev = "v17.1.1";
    hash = "sha256-XBH2pMDiHJNXrx90Lt0IcsbMFUM+X7GAHgiHpdlIdR4=";
  };
```